src/prop.c

```diff
@@ -84,6 +85,7 @@ prop_write_values
   uint32_t u32, opts;
   uint16_t u16;
   time_t tm;
+  int dyn_i;
```

Comment:
There's no need to declare it here, the declaration can be moved to where it's actually assigned/used

Reply:
If you search for 'tm' (variable above) you will see was defined and used the same. I followed the same 'code style'.

Reply:
@Flole998 do you have any updates in this request?

Reply:
@Flole998 this is my last attempt :( . Let me know if you have time to work on this this request.

Reply:
@Flole998 ^^^^^^^^